src/main/resources/project/Java.template

```diff
@@ -1,40 +1,40 @@
 package <package_name>;
 
 import ch.qos.logback.classic.Level;
-import <package_name>.generated.contracts.HelloWorld;
+import io.epirus.generated.contracts.HelloWorld;
 import io.epirus.web3j.Epirus;
 import io.epirus.web3j.gas.EpirusGasProvider;
 import io.epirus.web3j.gas.GasPrice;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.web3j.crypto.CipherException;
 import org.web3j.crypto.Credentials;
 import org.web3j.crypto.WalletUtils;
 import org.web3j.protocol.Network;
 import org.web3j.protocol.Web3j;
 import org.web3j.protocol.http.HttpService;
 import org.web3j.tx.gas.ContractGasProvider;
-import org.web3j.tx.gas.DefaultGasProvider;
-import org.web3j.tx.gas.StaticGasProvider;
 
-
-import java.io.File;
-import java.io.IOException;
-import java.math.BigInteger;
-import java.nio.file.Files;
 import java.nio.file.Paths;
 
+
 public class <project_name> {
 
     private static final Logger log = LoggerFactory.getILoggerFactory().getLogger("org.web3j.protocol.http.HttpService");
-    private static final String NODE_URL = "WEB3J_NODE_URL";
-    private static final Network deployNetwork = Network.valueOf(System.getenv().getOrDefault("EPIRUS_DEPLOY_NETWORK", "rinkeby").toUpperCase());
+    private static final boolean epirusDeploy = Boolean.parseBoolean(System.getenv().getOrDefault("EPIRUS_DEPLOY", "false"));
+    private static final Network deployNetwork = Network.valueOf(System.getenv().getOrDefault("WEB3J_DEPLOY_NETWORK", "rinkeby").toUpperCase());
+    private static final String walletPath = System.getenv("WEB3J_WALLET_PATH");
+    private static final String walletPassword = System.getenv().getOrDefault("WEB3J_WALLET_PASSWORD", "");
```

Comment:
What do you think of adding support of a raw private key too ?  We do that in OpenAPI project